manifests/common/plumgrid.pp

```diff
@@ -0,0 +1,69 @@
+# This class installs and configures Plumgrid Neutron Plugin.
+
+class openstack::common::plumgrid {
+  $is_controller = $::openstack::profile::base::is_controller
+  $controller_management_address = $::openstack::config::controller_address_management
+  $user = $::openstack::config::mysql_user_neutron
+  $pass = $::openstack::config::mysql_pass_neutron
+  $db_connection = "mysql://${user}:${pass}@${controller_management_address}/neutron"
+
+  if $is_controller {
```

Comment:
Instead of using if $is_controller, this whole section could just be moved to the neutron server profile, leaving only the part that is actually common to multiple profiles in the common::plumgrid class.